### DISCUSSION TITLE
Error made in the advanced-config.php

### DISCUSSION THREAD

**Discussion by Fraggle78** [JulianPrieber/littlelink-custom#781] [Questions]
Hello, I made a mistake in the advanced-config.php, which I edited in the ACP under Admin-Config. I wanted to add a link in the last part and now I have an error 500 and can't get rid of it.
When I go directly to the file in explorer, I can't find anything, it looks untouched. Do you have any tips?

error.log:
[Fri May 24 12:28:30.674359 2024] [proxy_fcgi:error] [pid 3739447:tid 139879176574656] [remote 185.37.250.59:57264] AH01071: Got error 'PHP message: PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:335\nStack trace:\n#0 /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(16): Illuminate\\Support\\Facades\\Facade::__callStatic()\n#1 /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(642): Illuminate\\Foundation\\Exceptions\\RegisterErrorViewPaths->__invoke()\n#2 /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(623): Illuminate\\Foundation\\Exceptions\\Handler->registerErrorViewPaths()\n#3 /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(554): Illuminate\\Foundation\\Exceptions\\Handler->renderHttpException()\n#4 /home/users/no404/www/linkstack/vendor/laravel/framework/src/Illuminate/Found...', referer: https://XXXXXXXX.de/admin/config

**Comment by JulianPrieber** [comment]
If you paste the file contents into something like this, it should point out the error: https://phpcodechecker.com/

In any case, you can just delete this file to restore it to its defaults.
